src/libs/ajax.js

```diff
@@ -1744,35 +1745,34 @@ const Dockstore = signal => ({
   listTools: (params = {}) => fetchDockstore(`api/ga4gh/v1/tools?${qs.stringify(params)}`).then(r => r.json())
 })
 
+const shouldUseDrsHub = !!fetchShouldUseDrsHub()
```

Comment:
```suggestion
const shouldUseDrsHub = !!getConfig().shouldUseDrsHub
```


As is, calling `fetchShouldUseDrsHub` will make a request to `<Terra URL>/true/undefined` or `<Terra URL>/false/undefined` depending on the value of `getConfig().shouldUseDrsHub`. This request will succeed because Terra responds with the app HTML for any unknown URL. Since `fetchShouldUseDrsHub()` returns a Promise, this will always be truthy.

Reply:
Thank you!